distribution/ddf-common/src/main/resources/common-bin.xml

```diff
@@ -33,6 +33,7 @@
         <exclude>etc/org.apache.karaf.log.cfg</exclude>
         <exclude>etc/org.ops4j.pax.web.cfg</exclude>
         <exclude>etc/users.properties</exclude>
+        <exclude>etc/startup.properties</exclude>
```

Comment:
This could cause problems in the future karaf upgrades if they change startLevels, add new bundles to startup, etc. Would it be better if we just modified the file rather than excluding it and providing our own?